### PR TITLE
Movie details cast crew sections

### PR DESCRIPTION
**Pull request by BoosterTech**
<MovieDetails/> page component created. <TopTileContainer/>, <MovieDetailsWrapper/>, sections Cast & Crew with default tiles added (first picture with <Navlink/> property to ProfileDetails page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { HashRouter, Route, Redirect, Switch } from "react-router-dom";
-import { toMovieListPage, toPeople, toProfile } from "./routes";
+import { toMovieDetails, toMovieListPage, toPeople, toProfile } from "./routes";
```
Comment (Pepko0) [inline src/App.js:2]: Nice Bro!

File: src/features/movieDetails/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          <PersonTile />
+          <PersonTile />
+          <PersonTile />
+          <PersonTile />
+          <PersonTile />
+          <PersonTile />
+          <PersonTile />
+          <PersonTile />
+          <PersonTile />
+          <PersonTile />
```
Comment (Pepko0) [inline src/features/movieDetails/index.js:38]: Good Idea

File: src/features/movieList/index.js [hunk 3]
```diff
@@ -67,19 +69,25 @@ export const MovieListPage = () => {
...
-                  }
-                  category={movieGenres}
-                  year={movie.release_date.substring(0, 4)}
-                  rate={movie.vote_average}
-                  vote={movie.vote_count}
-                />
+                <NavLink
+                  to={toMovieDetails()}
+                  key={moviesData.id}
+                  style={{ textDecoration: "none" }}
```
Comment (Arkadiusz0809) [inline src/features/movieList/index.js:75]: you can replace to styled.js ?

**Review by Pepko0 — APPROVED** [review]
Everything looks good

**Review by Arkadiusz0809 — APPROVED** [review]

**Pull request merged**